test/MC/RISCV/rvc.s

```diff
@@ -15,3 +15,30 @@ c.lw    a2, 4(a0)
 # CHECK-INST: c.sw    a5, 8(a3)
 # CHECK: encoding: [0x9c,0xc6]
 c.sw    a5, 8(a3)
+
+# CHECK-INST: c.j     -16
+# CHECK: encoding: [0xe5,0xbf]
+c.j     -16
+# CHECK-INST: c.jr    a7
+# CHECK: encoding: [0x82,0x88]
+c.jr    a7
+# CHECK-INST: c.jalr  a1
+# CHECK: encoding: [0x82,0x95]
+c.jalr  a1
+# CHECK-INST: c.beqz  a3, -8
+# CHECK: encoding: [0xf5,0xde]
+c.beqz  a3, -8
+# CHECK-INST: c.bnez  a5, -12
+# CHECK: encoding: [0xed,0xff]
+c.bnez  a5, -12
+
+.LBB0_2:
```

Comment:
I prefer to keep the fixup tests separate. See test/MC/RISCV/fixups.s. and fixups-diagnostics.s. Perhaps create fixups-compressed.s and fixups-compressed-diagnostics.s (which checks error messages for out of range immediates etc).

Reply:
Add fixups-compressed.s to extract c instruction's fixup testing from rvc.s.